gulp.action.sh

```diff
@@ -30,4 +35,17 @@ for i in "$@"; do
     esac
 done
 
-npx gulp "${TASK}" --platform="${PLATFORM}" --buildMode="${BUILD_MODE}"
\ No newline at end of file
+# if [[ "${ANDROID_KEY_STORE_PASSWORD}" != "" && "${BUILD_MODE}" == "debug" || "${PLATFORM}" != "android" ]]; then
+#     echo "TODO error text"
+#     exit 1
+# fi
+
+# if [[ "${BUILD_MODE}" == "release" && "${PLATFORM}" == "android" && "${ANDROID_KEY_STORE_PASSWORD}" != "" ]]; then
+#     echo "TODO error text"
+#     exit 1
+# fi
+
+npx gulp "${TASK}" \
+    --platform="${PLATFORM}" \
+    --buildMode="${BUILD_MODE}" \
+    --androidKeyStorePassword="${ANDROID_KEY_STORE_PASSWORD}"
```

Comment:
Flags are not good ways to pass secrets. They are often logged, show up on history and can be accessed by other processes. It will show up on the Github action logs!!!
Secrets must be either in a file, or env variable.

Reply:
Github censors these from the logs - it's the same with the match password for fastlane. But happy to move this into an env var purely